spec/models/article_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'rails_helper'
```

Comment:
Are you sure you need this require? Have you checked the other specs? What are you including in your `.rspec`?

Reply:
We saw that we have the `require rails_helper` code  included in the .rspec. 
Removed now.